c2cgeoportal/scaffolds/update/+package+/static/mobile/app/plugin/StatefulMap.js

```diff
@@ -117,6 +123,9 @@ Ext.define('App.plugin.StatefulMap', {
         if (q.map_x && q.map_y) {
             state.lonlat = [ q.map_x, q.map_y ];
             state.zoom = q.map_zoom;
+            if (parseInt(q.map_marker) === 1) {
```

Comment:
I just see that it also exists in the desktop app with the name 'map_crosshair':
https://github.com/camptocamp/cgxp/blob/master/core/src/script/CGXP/widgets/MapPanel.js#L152
I think that good to use the same name...
and we just do an `if (q.map_crosshair) ...`, than it can be '1', 'true' , ...